flink-table/flink-sql-parser/src/test/java/org/apache/flink/sql/parser/FlinkSqlParserImplTest.java

```diff
@@ -365,18 +365,18 @@ public void testCreateTableWithWatermarkStrategyFromSource() {
 	public void testCreateTableWithComplexType() {
 		check("CREATE TABLE tbl1 (\n" +
```

Comment:
I agree with Xuefu. All we need is actually one test `testCreateTableWithAllTypes` we could improve `testCreateTableWithComplexType` for that.

Reply:
The error happens during the validation phrase, not sql parsing, in `FlinkDDLDataTypesTest` we already test all the data types in the DDL. But i think add full types in `SqlToOperationTest` is more proper way to go.